packages/transit-vehicle-overlay/src/index.story.js

```diff
@@ -4,168 +4,287 @@ import { withA11y } from "@storybook/addon-a11y";
+    getTrackedVehicle,
+    updateTrackedVehicle
+  ] = utils.trackedVehicleState(fetchPattern, initVehicle, patternGeometry);
+  const [trackedVehicle, trackedRef] = getTrackedVehicle();
+  utils.linterIgnoreTheseProps(trackedRef);
+
+  // give action to the popup vehicle tracking button
+  VehiclePopup.defaultProps.setTracked = (vehicle, isTracked) => {
+    updateTrackedVehicle(vehicle, isTracked);
+    if (!isTracked) recenter(null, 0, 0); // clear out cached coords ... recenter on recently untracked vehicle
+  };
+
+  // record marker clicks (and more if no popup used)
+  const clickVehicle = (vehicle, isTracked) => {
+    setClicked(vehicle, isTracked);
+    // if no popup (e.g., popup has a tracking button), then track on marker click instead
+    if (!popup) {
+      updateTrackedVehicle(vehicle, isTracked);
+      if (!isTracked)
+        // clear recenter coords, so map recenters on repeated clicks of same marker
+        recenter(null, 0, 0);
+    }
+  };
+
+  // silly function used to change the arrival time in this example
+  function makeRandomDate() {
+    const secs = Date.now() % 379;
+    const prettyDate = formatDurationWithSeconds(secs);
+    return prettyDate;
+  }
+
+  // tooltip content callback function
+  CustomTooltip.defaultProps.getContent = (vehicle, isTracked) => {
+    utils.linterIgnoreTheseProps(isTracked);
+    const prettyDate = makeRandomDate();
+    let retVal;
+    if (vehicle && vehicle.routeShortName) {
+      retVal = `${vehicle.routeShortName} is arriving in ${prettyDate}`;
+    } else {
+      retVal = `Vehicle is arriving in ${prettyDate}`;
     }
+    return retVal;
+  };
 
-    return () => {
-      // before vehicle view component unmounts, clear the interval...
-      if (interval) {
-        clearInterval(interval);
-        interval = null;
-      }
-    };
-  }, []);
+  // if there's a popup, put perm tooltip on bottom of marker (since popup opens on top)
+  if (popup) CustomTooltip.defaultProps.direction = "bottom";
 
-  return justPresentationComponents(vehicleData, "#6013FE");
+  return (
+    <BaseMap
+      center={PORTLAND}
+      zoom={INITIAL_ZOOM_LEVEL}
+      onViewportChanged={onViewportChanged}
+    >
+      <TransitVehicleOverlay
+        name={null} // don't list this in the layer switcher
+        zoom={zoom}
+        vehicleList={vehicleData}
+        onVehicleClicked={clickVehicle}
+        selectedVehicle={trackedVehicle}
+        pattern={getRoutePattern()}
+        onRecenterMap={recenter}
+        MarkerSlot={ModeRectangles}
+        TooltipSlot={CustomTooltip}
+        PopupSlot={popup ? VehiclePopup : null}
+      />
+    </BaseMap>
+  );
 }
 
-function rtExample(name, visible) {
-  const geometryUrl = "https://newplanner.trimet.org/ws/ti/v0/index";
-  const vehicleUrl = "https://maps.trimet.org/gtfs/rt/vehicles/";
+/** with static data, show a simple version of the real-time transit vehicles layer */
+function realtimeExample(fetchVehicles, fetchPattern, markers) {
```

Comment:
I don't really understand the difference between realtimeExample and simpleExample based on the comment. Perhaps these function names could be more descriptive or the comments could be enhanced.